siddhi-testsuite/src/main/resources/archetype-resources/component/src/test/java/sample/blackbox/BlackBoxTestsOfTemperatureApp.java

```diff
@@ -0,0 +1,78 @@
+package sample.blackbox;
+
+import com.google.common.io.Resources;
+import io.siddhi.distribution.test.framework.SiddhiRunnerContainer;
+import io.siddhi.distribution.test.framework.util.NatsClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.output.OutputFrame;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import sample.AbstractTemperatureAlertTests;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class for black box testing of Temp-Alert-App. Run black-box testing by configuring the Siddhi Runner instances
+ * to communicate with the actual external systems.
+ *
+ * Description: Used for temperature monitoring and anomaly detection. Consumes events from a Nats topic,
+ *              filters the event under types 'monitored' and 'internal'.
+ *              Monitored events are then sent through a pattern and the matched events will be alerted to a Nats topic.
+ *              The internal events are persisted to a table.
+ */
+public class BlackBoxTestsOfTemperatureApp extends AbstractTemperatureAlertTests {
+    private static final Logger logger = LoggerFactory.getLogger(BlackBoxTestsOfTemperatureApp.class);
+
+    //Provide configurations for Nats and mysql environments.
```

Comment:
Rephrase the comment as **Update the Nats and MySql configurations as per your environment for Siddhi test framework to communicate with the said instances**